components/transactions/TransactionsDownloadInvoices.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/react-hooks';
+import { Download as IconDownload } from '@styled-icons/feather/Download';
+import gql from 'graphql-tag';
+import { groupBy, sumBy, truncate, uniq } from 'lodash';
+import moment from 'moment';
+import { FormattedMessage } from 'react-intl';
+
+import { formatCurrency } from '../../lib/currency-utils';
+import { useAsyncCall } from '../../lib/hooks/useAsyncCall';
+import { saveInvoice } from '../../lib/transactions';
+
+import Avatar from '../Avatar';
+import CollectivePicker, { FLAG_COLLECTIVE_PICKER_COLLECTIVE } from '../CollectivePicker';
+import { Box, Flex } from '../Grid';
+import PopupMenu from '../PopupMenu';
+import StyledButton from '../StyledButton';
+import StyledButtonSet from '../StyledButtonSet';
+import StyledSelect from '../StyledSelect';
+
+const FREQUENCIES = {
+  monthly: <FormattedMessage id="Frequency.Monthly" defaultMessage="Monthly" />,
+  yearly: <FormattedMessage id="Frequency.Yearly" defaultMessage="Yearly" />,
+};
+
+const getInvoicesQuery = gql`
+  query allInvoices($fromCollectiveSlug: String!) {
```

Comment:
```suggestion
  query TransactionsDownloadInvoices($fromCollectiveSlug: String!) {
```

We should aim to have descriptive and unique names.